redshift_sqlalchemy/dialect.py

```diff
@@ -249,6 +250,20 @@ def _get_column_info(self, *args, **kwargs):
 
         return column_info
 
+    def create_connect_args(self, *args, **kwargs):
+        default_args = {
+            'sslmode': 'verify-full',
```

Comment:
From http://docs.aws.amazon.com/redshift/latest/mgmt/connecting-ssl-support.html:

> Amazon Redshift does not support verify-full. For more information about sslmode options, see SSL Support in the PostgreSQL documentation.

So, if this is working, it's probably not doing what we think it's doing.